instrumentation/opentelemetry-instrumentation-urllib3/src/opentelemetry/instrumentation/urllib3/__init__.py

```diff
@@ -267,6 +268,7 @@ def _instrument(self, **kwargs):
                 name=HTTP_CLIENT_REQUEST_DURATION,
                 unit="s",
                 description="Duration of HTTP client requests.",
+                explicit_bucket_boundaries_advisory=HTTP_DURATION_HISTOGRAM_BUCKETS_NEW,
             )
             # http.client.request.body.size histogram
             request_size_histogram_new = create_http_client_request_body_size(
```

Comment:
Looks like the bucket boundaries will not be added to other histograms here

https://github.com/open-telemetry/opentelemetry-python/blob/c9ad4bc2cefc64a918f0cda3cf76c64379c87aea/opentelemetry-semantic-conventions/src/opentelemetry/semconv/_incubating/metrics/http_metrics.py#L80

Reply:
These are generated from specs but yeah would be nice to them automatically to `create_http_client_request_duration` and `create_http_server_request_duration`

Reply:
Ah but if you really want to reference the body request size this PR is meant to update only the duration metrics that are expressed in seconds while the default buckets are more suitable for milliseconds. I think the default buckets fits better than these duration specific for bytes but if I'm missing something feel free to open an issue.

Reply:
Filed this https://github.com/open-telemetry/opentelemetry-python/issues/4598 for the generic issue of being able to have helpers with proper buckets